Resources/public/js/views/ez-languageselectionboxview.js

```diff
@@ -0,0 +1,418 @@
+        },
+
+        /**
+         * Returns list of languages that are available but content doesn't have translation
+         * in these languages
+         *
+         * @protected
+         * @method _getNewTranslations
+         * @return {Array}
+         */
+        _getNewTranslations: function () {
+            var translations = Y.Object.keys(this.get('availableTranslations')),
+                existingTranslations = this.get('existingTranslations');
+
+            return Y.Array.reject(translations, function (value) {
+                return existingTranslations.indexOf(value) >= 0;
+            });
+        },
+    }, {
+        ATTRS: {
+            /**
+             * Title of the languageSelectionBox
+             *
+             * @attribute title
+             * @default ""
+             * @type {String}
+             */
+            title: {
+                value: "",
+            },
+
+            /**
+             * confirm event handler
+             *
+             * @attribute confirmHandler
+             * @type {Function|Null}
+             * @default null
+             */
+            confirmHandler: {
+                value: null,
+            },
+
+            /**
+             * cancel event handler
+             *
+             * @attribute cancelHandler
+             * @type {Function|Null}
+             * @default null
+             */
+            cancelHandler: {
+                value: null,
+            },
+
+            /**
+             * List of available new translations
+             *
+             * @attribute newTranslations
+             * @type {Array}
+             */
+            newTranslations: {
+                getter: '_getNewTranslations'
+            },
+
+            /**
+             * Already existing translations of content
+             *
+             * @attribute existingTranslations
+             * @type {Array|Null}
+             * @default null
+             */
+            existingTranslations: {
+                value: null
+            },
+
+            /**
+             * Enables or disables possibility of basing new translation on already existing one
+             *
+             * @attribute canBaseTranslation
+             * @type {Boolean}
+             * @default true
+             */
+            canBaseTranslation: {
+                value: true
+            },
+
+            /**
+             * Defines if new translation will be basing on already existing one
+             *
+             * @attribute baseTranslation
+             * @type {Boolean}
+             * @default false
+             */
+            baseTranslation: {
+                value: false
+            },
+
+            /**
+             * Selected language from new translations list
+             *
+             * @attribute selectedNewTranslation
+             * @type {string|Null}
+             * @default null
+             */
+            selectedNewTranslation: {
```

Comment:
this attribute and `selectedBaseTranslation` should be called `Language` not `Translation` as the language selection box is meant to allow to select a language not only to translate something.